src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -138,7 +138,7 @@ export class SearchInterface extends RootComponent implements IComponentBindings
      * > {@link Facet.options.dropdownHeaderLabel} and {@link Tab.options.dropdownHeaderLabel}).
      * >
      * > Furthermore, it is possible to specify the pixel threshold at which Facet components will go in responsive
-     * > mode (see {@link Facet.options.responsiveBreakpoint}.
+     * > mode (see {@link Facet.options.responsiveBreakpoint}).
```

Comment:
We modified that, but the doc has not been adjusted. 

The breakpoint is no longer specified per component. It's a global setting, on the search interface component (SearchInterface.responsiveComponents).

So this should be modified.